code/modules/antagonists/changeling/changeling.dm

```diff
@@ -97,22 +97,25 @@
 	handle_clown_mutation(owner.current, removing=FALSE)
 	. = ..()
 
-/datum/antagonist/changeling/proc/reset_properties()
+/datum/antagonist/changeling/proc/reset_properties(reset_abilities = FALSE)
```

Comment:
Is there a way to call it with false?

Reply:
That's called by /datum/antagonist/changeling/on_removal()

Reply:
It doesn't really matter if we should keep the point when removing ling.
Can't you just hard code it?

Reply:
Sure